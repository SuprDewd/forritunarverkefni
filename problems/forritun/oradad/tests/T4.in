-24 -97 -36 86 77
