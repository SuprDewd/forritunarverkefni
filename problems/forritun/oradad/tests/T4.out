-36 -97 -24 77 86
